tests/support.rs

```diff
@@ -0,0 +1,92 @@
+extern crate env_logger;
```

Comment:
The idiom is to put this file at `support/mod.rs`. That way it doesn't try to compile it as a test.